test/e2e/webdriver/chrome.js

```diff
@@ -34,11 +40,21 @@ class ChromeDriver {
     } else {
       args.push('--log-level=3');
     }
+    if (process.env.SELENIUM_HEADLESS) {
+      args.push('--headless=new');
+    }
```

Comment:
cool 

https://developer.chrome.com/docs/chromium/new-headless
https://www.selenium.dev/blog/2023/headless-is-going-away/